ci_framework/roles/libvirt_manager/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_libvirt_manager"
+
+cifmw_libvirt_manager_basedir: "{{ cifmw_basedir | default( ansible_user_dir ~ '/ci-framework') }}"
+cifmw_libvirt_manager_enable_virtualization_module: false
+cifmw_libvirt_manager_user: "{{ ansible_user | default(lookup('env', 'USER')) }}"
+cifmw_libvirt_manager_images_url: https://cloud.centos.org/centos/9-stream/x86_64/images
+
+cifmw_libvirt_manager_configuration:
+  networks:
+    - default
+  vms:
+    compute:
+      amount: 2
+      image_url: "{{ cifmw_libvirt_manager_images_url }}/CentOS-Stream-GenericCloud-9-20210830.0.x86_64.qcow2"
+      sha256_image_name: 99df867be3300ba5292e274acd149dac1dc784f9db4c30bdf8e3c89abd639976
+      disk_file_name: "{{ cifmw_libvirt_manager_basedir }}/artifacts/centos-9-stream.qcow2"
```

Comment:
can't we make it automatic? iirc cloud.centos.org presents the checksum as <image_name>.<checksum>

Reply:
probably a good thing to keep close, especially if we're to get some more automated way to get the "latest" image.